src/components/plots/areaPlot.ts

```diff
@@ -127,9 +155,11 @@ export module Plots {
 
     protected _propertyProjectors(): AttributeToProjector {
       var propertyToProjectors = super._propertyProjectors();
-      propertyToProjectors["d"] = this._constructAreaProjector(Plot._scaledAccessor(this.x()),
-                                                               Plot._scaledAccessor(this.y()),
-                                                               Plot._scaledAccessor(this.y0()));
+      if (this._projectorsReady()) {
```

Comment:
`_propertyProjectors()` is theoretically called after `x` / `y` is set.  Can you look into when this is called?